JetNews/app/src/main/java/com/example/jetnews/ui/theme/Color.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.jetnews.ui.theme
+
+import androidx.ui.graphics.Color
+
+val red200 = Color(0xfff297a2)
```

Comment:
Maybe these should be PascalCased? val Red200, etc.

Reply:
Good call.